runpod/serverless/utils/rp_upload.py

```diff
@@ -14,13 +14,14 @@
 from typing import Optional, Tuple
 
 import boto3
-from PIL import Image
+from PIL import Image, UnidentifiedImageError
 from boto3 import session
 from boto3.s3.transfer import TransferConfig
 from botocore.config import Config
 from tqdm_loggable.auto import tqdm
 
 
+
```

Comment:
Everything else looks good, I almost feel bad asking for the one change to get rid of this extra blank line :)